scripts/change-certificates-structure.mjs

```diff
@@ -0,0 +1,20 @@
+
+import * as fs from 'fs'
```

Comment:
Check if we could use an `mts` or `ts` file extension

Reply:
Using Typescript in this file would lead to constant type errors since the script is specifically altering the object's structure, so I would suggest to keep it as .mjs. Do you agree?